relay-filter/src/config.rs

```diff
@@ -328,15 +482,208 @@ impl FiltersConfig {
     }
 }
 
+/// Configuration of generic filters combined from project and global config.
+///
+/// See [`CombinedFiltersConfigIter`] for details on how to iterate easily
+/// through the applicable filters.
+#[derive(Copy, Clone, Debug)]
+pub(crate) struct CombinedFiltersConfig<'a> {
```

Comment:
so currently we have used the `Dynamic` prefix for these combined global and project types. I'm fine with changing to `Combined` but then we should change the other instances too.